ovs/lib/albacontroller.py

```diff
@@ -106,18 +114,36 @@ def add_preset(alba_backend_guid, name, compression, policies):
         preset = {'compression': compression,
                   'object_checksum': {'default': ['crc-32c'], 'verify_upload': True, 'allowed': [['none'], ['sha-1'], ['crc-32c']]},
                   'osds': ['all'],
-                  'fragment_encryption': ['none'],
                   'fragment_size': 1048576,
                   'policies': policies,
                   'fragment_checksum': ['crc-32c'],
                   'in_use': False,
                   'name': name}
+
+        if encryption in ['aes-cbc-256']:
+            encryption_key = ''
+            while len(encryption_key) != 32:
+                encryption_key = ''.join(random.choice(chr(random.randint(32, 126))) for _ in range(32))
+            temp_key_file = tempfile.mktemp()
+            with open(temp_key_file, 'wb') as f:
+                f.write(encryption_key)
+                f.flush()
+                f.close()
```

Comment:
Single char variable: 'f'   :(
f.close() is not necessary in a 'with' statement, the __exit__ function will close the file